client/src/api/notes.js

```diff
@@ -0,0 +1,47 @@
+export function getNote(noteId) {
+  return fetch(`/api/notes/${noteId}`, {
+    method: 'GET',
+  })
+    .then((response) => {
+      if (response.status !== 200) {
```

Comment:
Are you sure you want to throw an error for the other cases like `301`?
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
Check **Checking that the fetch was successful**

```suggestion
      if (!response.ok) {
```

(happens multiple times)